pkg/innerring/innerring.go

```diff
@@ -661,6 +690,7 @@ func New(ctx context.Context, log *zap.Logger, cfg *viper.Viper) (*Server, error
 		AlphabetSyncHandler: alphaSync,
 		NodeValidator: nodevalidator.New(
 			addrvalidator.New(),
+			subnetValidator,
```

Comment:
Why not last?

Reply:
not sure what validator is more expensive

Reply:
But the code says you thought that subnet is more expensive, even considering that it refers to the Neo RPC node as opposed to the locode.

Reply:
because i thought so

ok, moved at the end